Projekty/Projekt1/Gryszkiewicz_Chylak/Wstępny_Model/model_wstepny.ipynb

```diff
@@ -0,0 +1,2038 @@
+{
```

Comment:
<p>Bardzo dobrze wykonany KM2. Rzetelna inżynieria cech.</p><ul><li>Można się zastanowić nad skalowaniem zmiennych.</li><li>Zabrakło sekcji podsumowanie.</li><li>Warto byłoby dodać macierz pomyłek.</li></ul><p>W ramach KM3 warto byłoby wybrać optymalne parametry modeli (na zajęciach zostanie omówione strojenie).</p><p> </p><p>Uwaga na przyszłość:</p><ul><li>Ciekawa byłaby analiza co jest "wyjątkowego" w obserwacjach, na których finalny model się myli - zajmuje się tym dziedzina Interpretowalnego Uczenia Maszynowego (być może będzie kiedyś okazja wspomnieć o wykorzystywanych algorytmach na zajęciach)</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>